src/app/[path]/forms/[form]/page.tsx

```diff
@@ -1,4 +1,3 @@
-// Example: courtformsonline.org/ma/forms/[form-slug]
```

Comment:
I'd prefer to leave this comment in place. It helps me find my way around the various page.tsx templates. But it's not essential.